pkg/cmd/pipeline/list_test.go

```diff
@@ -1,24 +1,83 @@
 package pipeline
 
 import (
+	"bytes"
+	"strings"
 	"testing"
 
+	"github.com/google/go-cmp/cmp"
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/client/clientset/versioned"
 	"github.com/tektoncd/pipeline/test"
 	tb "github.com/tektoncd/pipeline/test/builder"
 )
 
+type TestParams struct {
```

Comment:
can we put these in some common package to reuse in other tests

Reply:
yes, done!